includes/init.php

```diff
@@ -347,9 +363,17 @@ function print_trailer( $include_nav_links = true, $closeDb = true,
   }
 
   return $ret . '
-<!-- ' . $GLOBALS['PROGRAM_NAME'] . '     ' . $GLOBALS['PROGRAM_URL'] . ' -->
-' // Adds an easy link to validate the pages.
-  . ( $DEMO_MODE == 'Y' ? '
+<!-- ' . $GLOBALS['PROGRAM_NAME'] . '     ' . $GLOBALS['PROGRAM_URL'] . ' -->' .
+    ( $ALLOW_HTML_DESCRIPTION !== 'Y' ? '' :
+    /* Your choices here are "basic", "standard" or "full". */ '
+    <script src="//cdn.ckeditor.com/4.6.0/basic/ckeditor.js"></script>
```

Comment:
No. 4.6.0 is just the one I found when I was actually trying to work on something.

I'd like to see a local system check for ckeditor, phpmailer, recaptcha, smarty, etc.
I'd also like to give the users the option of not using any third party code or letting them install any that they want to use somewhere outside the WebCalendar directory. Like "http://mysite.com/common/" or something. There seems to be a lot of people who already have some of these things installed on their sites. We should try to use what they have, if possible.

I'd like to see everything we don't maintain as remote requests. :) But, it doesn't actually hurt much to have local copies, too.